tests/eclient/testdata/port_forward.txt

```diff
@@ -0,0 +1,94 @@
+# Tests for port forward connectivity between applications
+
+{{$test_msg := "Port forward tests"}}
+{{define "ssh"}}ssh -o ConnectTimeout=10 -o StrictHostKeyChecking=no -i {{EdenConfig "eden.tests"}}/eclient/image/cert/id_rsa root@{{end}}
+
+[!exec:bash] stop
+[!exec:sleep] stop
+[!exec:ssh] stop
+[!exec:chmod] stop
+
+exec chmod 600 {{EdenConfig "eden.tests"}}/eclient/image/cert/id_rsa
+
+# Starting of reboot detector with 2 reboots limit
+! test eden.reboot.test -test.v -timewait 10m -reboot=0 -count=2 &
```

Comment:
This might apply to other tests, but shouldn't we run the reboot detector with a limit of zero in general since we normally don't reboot EVE in the tests? @giggsoff ?

Reply:
Well, here we should set reboot to zero (no initiate reboot), count to one (wait for one reboot) and timewait to any large time (wait for reboot for the whole test time, it will exited on tests end). This detector runs in background (& in the end) and expected to be failed (! in start of it). If it detects reboot (count of reboots), it will ends with exit code 0, which is unexpected (we put ! in the start of it), so, it will broke the exicution.

Reply:
OK, I missed the ! in start of it.